lib/galaxy/managers/configuration.py

```diff
@@ -61,6 +61,7 @@ def _defaults_to(default):
             'allow_user_creation'               : _defaults_to(False),
             'use_remote_user'                   : _defaults_to(None),
             'enable_oidc'                       : _defaults_to(False),
+            'oidc'                              : _defaults_to(self.app.config.oidc),
```

Comment:
probably a more intuitive name for this variable would be `oidc_idps`, or `oidc_backends`, or `oidc_identity_providers`.